Services/data/devicemodels/elevator-01.json

```diff
@@ -44,5 +44,19 @@
                 }
             }
         }
-    ]
+    ],
```

Comment:
Also update the reported properties to include Type, Firmware and Model

Reply:
I'm deliberately keeping things as is. As we get to actually implementing these methods we should look at it. I believe we already have a PBI for this.